megamek/src/megamek/client/ui/swing/boardview/TurnDetailsOverlay.java

```diff
@@ -67,10 +57,6 @@ protected void gameTurnOrPhaseChange() {
         super.gameTurnOrPhaseChange();
     }
 
-    @Override
-    protected void setVisibilityGUIPreference(boolean value) {
```

Comment:
Where did the setVisibilityGUIPreference logic get moved to?

Reply:
and does this still respond to GUIP changes with the IPreferenceChangeListener removed?

Reply:
not finished yet! I first moved the reaction into BoardView but I've since changed my viewpoint. I'll move it back into the Overlay. The logic for this is that the menu item for any of the overlays only changes the GUIP setting and everything else happens as a preferenceChangeEvent reaction. We don't have many static event generators but this one we must make use of :)